main.tscn

```diff
@@ -23,48 +23,63 @@ metadata/_edit_group_ = true
 
 [node name="Dome" parent="Domes" instance=ExtResource("1_o7dif")]
 position = Vector2(144, 109)
+resource_type = 0
 
 [node name="Dome2" parent="Domes" instance=ExtResource("1_o7dif")]
 position = Vector2(366, 203)
+resource_type = 0
 
 [node name="Dome3" parent="Domes" instance=ExtResource("1_o7dif")]
 position = Vector2(388, 70)
+resource_type = 0
 
 [node name="Dome4" parent="Domes" instance=ExtResource("1_o7dif")]
```

Comment:
@avelitj12 idk what this is about, do you keep getting this too?

Reply:
Yeah, just weirdness I ignore